tests/testthat/test-survival_reg-flexsurv.R

```diff
@@ -1,25 +1,28 @@
 library(testthat)
 
-test_that("flexsurv execution", {
-  expect_error(
-    res <- survival_reg(dist = "lognormal") %>%
```

Comment:
A question more for my own understanding than about correctness:

There are four removals of this `dist = "lognormal"` argument but no additions including it. Are we still testing this somewhere / do we need to? Is this a default now?

Reply:
yes, a different test (the one on survival time predictions) still uses a non-default distribution 👍